chapters/05-inductive.php

```diff
@@ -525,7 +525,7 @@
+		<p>When dealing with an incomplete knowledge graph, it is not immediately clear how to define negative edges. A common heuristic – also used for knowledge graph embeddings – is to adopt a Partial Completeness Assumption (PCA)&nbsp;<?php echo $references->cite("GalarragaTHS13"); ?>, which considers the set of positive edges to be those contained in the data graph, and the set of negative examples to be the set of all edges <?php echo gedge("\(x\)","\(p\)","\(y'\)"); ?> not in the graph but where there exists a node <span class="gnode">\(y\)</span> such that <?php echo gedge("\(x\)","\(p\)","\(y\)"); ?> is in the graph. Taking Figure&nbsp;<?php echo ref("fig:airports"); ?>, an example of a negative edge under PCA would be <?php echo gedge("SCL","flight","ARI"); ?> (given the presence of <?php echo gedge("SCL","flight","LIM"); ?>); conversely, <?php echo gedge("SCL","domestic&nbsp;flight","ARI"); ?> is neither positive nor negative. The PCA confidence measure is then the ratio of the support to the number of entailments in the positive or negative set&nbsp;<?php echo $references->cite("GalarragaTHS13"); ?>. For example, the support for the rule <?php echo sedge("?x","domestic&nbsp;flight",NULL,"?y","gvar"); ?> \(\Rightarrow\) <?php echo sedge("?y","domestic&nbsp;flight",NULL,"?x","gvar"); ?> is \(2\) (since it entails <?php echo gedge("IQQ","domestic&nbsp;flight","ARI"); ?> and <?php echo gedge("ARI","domestic&nbsp;flight","IQQ"); ?> in the graph, which are thus positive edges), while the confidence is \(\frac{2}{2} = 1\) (noting that <?php echo gedge("SCL","domestic&nbsp;flight","ARI"); ?>, though entailed, is neither positive nor negative, and is thus ignored by the measure). The support for the rule <?php echo sedge("?x","flight",NULL,"?y","gvar"); ?> \(\Rightarrow\) <?php echo sedge("?y","flight",NULL,"?x","gvar"); ?> is analogously 4, while the confidence is \(\frac{4}{5} = 0.8\) (noting that <?php echo gedge("SCL","flight","ARI"); ?> is a negative edge).</p>

```

Comment:
I suggest swapping `y` and `y'` so that `y` is introduced before its prime counterpart.

Reply:
Changed accordingly in https://github.com/Knowledge-Graphs-Book/HTML-Book/pull/57/commits/5891475bb7d170b247a5b03ed5ff655056a677e3